configuration/configs/test.json

```diff
@@ -273,7 +273,7 @@
   "agent": {
     "ethereum": {
       "rpcStyle": "ethereum",
-      "db": "/usr/share/nomad",
+      "db": "db",
```

Comment:
Need more context on this

Reply:
Yeah just wanted to change this to `db` so we can run the script locally (as I am doing for local testing). `/usr/share` is a protected folder you can't write to without mounting or smth non-local-friendly